stacks/play.prx.org.yml

```diff
@@ -212,6 +212,40 @@ Resources:
             DNSName: !GetAtt PlayCloudFrontDistribution.DomainName
             HostedZoneId: Z2FDTNDATAQYW2
   # CloudFront
+  PlayCloudFrontCachePolicy:
+    Type: AWS::CloudFront::CachePolicy
+    Properties:
+      CachePolicyConfig:
+        Comment: !Sub Policy for ${AWS::StackName}
+        DefaultTTL: 86400
```

Comment:
These are the defaults, right?  So not actually changing anything.  And I think all the play.prx.org endpoints set their own TTLs anyways.

Reply:
They are the defaults but for some reason those properties are required in the new format